### ISSUE TITLE
awxkit: no source distribution files available for this release on pypi.org

### ISSUE THREAD

**Issue by computersalat** [ansible/awx#14744]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### Feature type

New Feature

### Feature Summary

Source Distribution Files are missing here: [awxkit 23.6.0](https://pypi.org/project/awxkit/#files)

### Select the relevant components

- [ ] UI
- [ ] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [X] Other

### Steps to reproduce

try to download the awxkit Source files:
```
wget https://files.pythonhosted.org/packages/source/a/awxkit/awxkit-23.6.0.tar.gz
```

### Current results

404 - File Not Found
```
--2024-01-04 12:29:26--  https://files.pythonhosted.org/packages/source/a/awxkit/awxkit-23.6.0.tar.gz
Auflösen des Hostnamens files.pythonhosted.org (files.pythonhosted.org)… 146.75.120.223, 2a04:4e42:8e::223
Verbindungsaufbau zu files.pythonhosted.org (files.pythonhosted.org)|146.75.120.223|:443 … verbunden.
HTTP-Anforderung gesendet, auf Antwort wird gewartet … 404 Not Found
2024-01-04 12:29:27 FEHLER 404: Not Found.
```

### Sugested feature result

There should be a file for download. Some people maybe want to make use of it for building an RPM/deb package.
Not everyone is fancy with **_pip_**

### Additional information

Maybe that creation and upload process got lost in the past.
Would be nice to revive that process again and provide source files (tar.gz).
Thank you

**Comment by jbradberry** [comment]
Noting this here for reference:

Looking back through pypi's history, 19.4.0 (released 2 Oct 2021 https://pypi.org/project/awxkit/19.4.0/#files) was the last one to have a source file distribution.

The next one, 20.0.1 (released 8 Mar 2022 https://pypi.org/project/awxkit/20.0.1/#files), does not have it.

Update: This time-period is when we switched the release project to using Github actions (https://github.com/ansible/awx/releases/tag/20.0.1).  It was probably broken by mistake.

**Comment by computersalat** [comment]
> Noting this here for reference:
> 
> Looking back through pypi's history, 19.4.0 (released 2 Oct 2021 https://pypi.org/project/awxkit/19.4.0/#files) was the last one to have a source file distribution.
> 
> The next one, 20.0.1 (released 8 Mar 2022 https://pypi.org/project/awxkit/20.0.1/#files), does not have it.
> 
> Update: This time-period is when we switched the release project to using Github actions (https://github.com/ansible/awx/releases/tag/20.0.1). It was probably broken by mistake.

Thank you for looking in to this and solving :-)

**Comment by fosterseth** [comment]
implemented in #14757

**Comment by jbradberry** [comment]
23.7.0 (https://pypi.org/project/awxkit/#files) successfully got the source bundle.
